### ISSUE TITLE
Kayn Ult Audio Bug

### ISSUE THREAD

**Issue by Nimzuie** [R3nzTheCodeGOD/R3nzSkin#247]
When ulting with Kayn using any custom skin there's a pretty high chance the ult sound effect will permanently play for the rest of the game until restarted/exited

**Comment by alphaaced** [comment]
yes on odyssey kayn and others. Very loud and annoying pls fix.

**Comment by alphaaced** [comment]
yes on odyssey kayn and others. Very loud and annoying pls fix if possible @R3nzTheCodeGOD .
